csrc/index_compute.cpp

```diff
@@ -654,7 +654,6 @@ IndexCompute::IndexCompute(
       zero_domains_(std::move(zero_domains)),
       zero_merged_in_(std::move(zero_merged_in)),
       contig_ids_{contig_finder.contigIDs()},
-      root_to_indexed_id_{contig_finder.rootToIndexedID()},
```

Comment:
Why was this removed?

Reply:
This is never used, `root_to_indexed_id_` is also removed.